server/models/Articles.js

```diff
@@ -0,0 +1,9 @@
+
+const nameSchema = new mongoose.Schema({
+  title: String,
+  fullContent: String
+});
+const article = mongoose.model("article", nameSchema);
```

Comment:
Can we make this capitalised? "Article"
Because it refers to an entity, a thing, so we usually capitalise them.